core/src/datayoga_core/block.py

```diff
@@ -6,13 +6,15 @@
 import sys
 from abc import ABCMeta, abstractmethod
 from os import path
-from typing import Any, Dict, List, Optional
+from typing import Any, Dict, Generator, List, Optional, TypedDict
 
 from datayoga_core import utils
 from datayoga_core.context import Context
 from datayoga_core.result import BlockResult
 from jsonschema import validate
 
+Message = TypedDict("Message", dict(msg_id=str, value=Dict[str, Any]))
```

Comment:
linter:
Consider using '{"msg_id": str, "value": Dict[str, Any]}' instead of a call to 'dict'.pylint(use-dict-literal)
Expected dict or keyword parameter as second parameterPylance

Reply:
✅